scripts/docker-compose-scylla.yml

```diff
@@ -11,15 +11,25 @@ networks:
 
 services:
   gemini-oracle:
-    image: scylladb/scylla:5.2.2
+    image: scylladb/scylla:${SCYLLA_ORACLE_VERSION:-5.2.19}
     container_name: gemini-oracle
-    command: --smp 2 --memory 512M --api-address 0.0.0.0
+    entrypoint: /entry.sh
+    ports:
+      - "9042:9042"
+    volumes:
+      - ${PWD}/scylla-entry.sh:/entry.sh
+    command: --smp 2 --memory 1024M --api-address 0.0.0.0 --cpus=1
```

Comment:
Can you please provide details on why do you think that `--cpus=1` help here.

Reply:
Mistake, left after some tests, will be removed